thirdeye-ui/src/app/locale/languages/en-us.json

```diff
@@ -705,6 +705,7 @@
         "our-new-feature-sets-up-your-alert-with-the-parameters": "Our new feature sets up your alert with the parameters that best fit your data. (currently in beta)",
         "page-not-found": "Requested page could not be found",
         "percentage-rule-algorithm-description": "Compare current timeseries to a baseline. If the percentage change is above a certain threshold, detects an anomaly.",
+        "pinot-broker-error": "Raise a ticket with support team to fix Pinot broker listening issues.",
```

Comment:
Add something like: 
"Pinot query failed"? 
Also we should still be able to see the query, it really helps when debugging. 
I think the simplest way would be to have a `<details>` that make it possible to show the `message`  fields returned by the backend.